### PR TITLE
Fix serialized property in mustache templating

### PR DESCRIPTION
**Pull request by fgravin**
`serialized` property is added to template input feature before the rendering.
eg.
```js
{
  name: 'Paris',
  code:  75000,
  dep: 75
}
```
Somehow, this property (which was initially a key/value pairs object containing all features properties) became a full GeoJSON object, including the geometry.

This PR aims to fix it, and refactor a bit the creation of both  `serialized` and `fields_kv` bbjects.

### CODE REVIEW

File: js/info.js [hunk 1]
```diff
@@ -679,28 +678,30 @@ var info = (function () {
...
-              return fields_kv;
-            }
+                var properties = extractFeaturePropertiesFn(this)
+                return Object.entries(properties).map(([key, value]) => {
+                    return {key, value}
+                })
+            };
             feature.setProperties({'fields_kv': fields_kv});
             // add a serialized version of the object so it can easily be passed through HTML GET request
             // you can deserialize it with `JSON.parse(data)` when data is the serialized data
```
Comment (Gaetanbrl) [inline js/info.js:702]: Return error with `JSON.parse(data)`.
Needs `JSON.parse(decodeURIComponent(feature.getProperties().serialized()))` to be readable.

**Review by Gaetanbrl — APPROVED** [review]
Thanks. LGTM.

image

**Comment by Gaetanbrl** [comment]
Oh ! @fgravin you target the wrong branch.

Please target develop.

thanks.

**Review by Gaetanbrl — CHANGES_REQUESTED** [review]
This PR target the wrong branch.

Please, target develop as expected / explain  : 

https://mviewerdoc.readthedocs.io/fr/latest/doc_contrib/git.html?highlight=workflow

Thanks.

File: js/info.js (hunk 1 above)
Comment (fgravin) [reply js/info.js:702]: Do you want me to change the legacy comment ?

**Comment by fgravin** [comment]
Hi @Gaetanbrl thanks for the review.
Yes `fields_kv` keeps working the same, only the property `serialized` has been fixed.

I changed the target to `develop`, sorry for that.

**Comment by fgravin** [comment]
> I changed the target to `develop`, sorry for that.

Need to resolve the conflicts now :)
I ping you when it's done

File: js/info.js (hunk 1 above)
Comment (Gaetanbrl) [reply js/info.js]: Why not. Thanks.

**Comment by fgravin** [comment]
Hi @Gaetanbrl, I have rebased on `develop` and eveything looks fine.
I also update the comment as you expected.

Thanks

**Comment by spelhate** [comment]
Thank's @Gaetanbrl for the review.

**Pull request merged**